apps/dapp-console-api/src/models/addresses.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  index,
+  jsonb,
+  pgTable,
+  timestamp,
+  uniqueIndex,
+  uuid,
+  varchar,
+} from 'drizzle-orm/pg-core'
+import type { Address } from 'viem'
+
+import { entities } from './entities'
+
+enum AddressState {
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+}
+
+export const addresses = pgTable(
+  'addresses',
+  {
+    createdAt: timestamp('created_at', { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+    updatedAt: timestamp('updated_at', { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+    id: uuid('id').defaultRandom().primaryKey(),
+    entityId: uuid('entity_id').references(() => entities.id),
+    address: varchar('address').$type<Address>().notNull(),
+    verifications: jsonb('verifications').default({}).notNull(),
+    state: varchar('state')
+      .$type<AddressState>()
+      .default(AddressState.ACTIVE)
+      .notNull(),
+  },
+  (table) => {
+    return {
+      entitiyIdAddressIdx: uniqueIndex().on(table.entityId, table.address),
+      addressIdx: index('address_idx').on(table.address),
```

Comment:
We should avoid passing in strings to the index functions with drizzle. Index names exist on the global namespace across the database so this will create an index called `address_idx` which will conflict with other tables if they attempt to call it that. When we don't pass a string into the function drizzle will namespace it with the table name automatically so it won't collide.

Reply:
makes sense, good catch